manifest/com.TheJebForge/BoneReferenceHelper/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "BoneReferenceHelper",
+	"id": "com.TheJebForge.BoneReferenceHelper",
+	"description": "Adds a lot of useful things for transferring bone references of skinned mesh renderers",
+	"category": "Inspectors",
+	"sourceLocation": "https://github.com/TheJebForge/BoneReferenceHelper",
+	"versions": {
+		"2.0.5": {
+			"releaseUrl": "https://github.com/TheJebForge/BoneReferenceHelper/releases/tag/v2.0.5",
+			"artifacts": [{
+				"url": "https://github.com/TheJebForge/BoneReferenceHelper/releases/download/v2.0.5/BoneReferenceHelper.dll",
```

Comment:
nitpick, preferably the `new Harmony("com." + this.Author + "." + this.Name).PatchAll();` just uses the static id instead of constructing it. Looks good otherwise